architecture/juce/juce-plugin.cpp

```diff
@@ -627,6 +652,10 @@ void FaustPlugInAudioProcessor::prepareToPlay (double sampleRate, int samplesPer
     fDSP->instanceClear();
     
 #endif
+#ifdef MAGIC_LEVEL_SOURCE
+    magicState.prepareToPlay(sampleRate, samplesPerBlock);
+    outputMeter->setupSource(2, sampleRate, 500, 200);
```

Comment:
Would be better to use `getMainBusNumInputChannels()` and `getMainBusNumOutputChannels()` for the output

Reply:
Will update.